### PR TITLE
Update swagger security schemes

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/certificates/Query.API/API.IntegrationTests/SwaggerTests.SwaggerDoc_20230101_VerifiesFullBearerSecuritySchemeStructure.verified.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿{
+      "Bearer": {
+        "type": "http",
+        "description": "JWT Authorization header using the Bearer scheme. \r\n\r\n Enter 'Bearer' [space] and then your token in the text input below.\r\n\r\nExample: \"Bearer 1safsfsdfdfd\"",
```
Comment (ckr123) [inline domains/certificates/Query.API/API.IntegrationTests/SwaggerTests.SwaggerDoc_20230101_VerifiesFullBearerSecuritySchemeStructure.verified.txt:4]: Are these \r n and needed?

File: domains/certificates/Query.API/API/Query.API/Swagger/ConfigureSwaggerOptions.cs [hunk 2]
```diff
@@ -31,12 +22,14 @@ public void Configure(SwaggerGenOptions options)
...
-            Type = SecuritySchemeType.ApiKey,
+            Type = SecuritySchemeType.Http,
             Scheme = "Bearer",
             BearerFormat = "JWT",
             In = ParameterLocation.Header,
             Description =
-                "JWT Authorization header using the Bearer scheme. \r\n\r\n Enter 'Bearer' [space] and then your token in the text input below.\r\n\r\nExample: \"Bearer 1safsfsdfdfd\""
+                "JWT Authorization header using the Bearer scheme." +
+                " \r\n\r\n Enter 'Bearer' [space] and then your token in the text input below." +
+                "\r\n\r\nExample: \"Bearer 1safsfsdfdfd\""
```
Comment (ckr123) [inline domains/certificates/Query.API/API/Query.API/Swagger/ConfigureSwaggerOptions.cs:32]: Are these \r n and needed?

File: domains/certificates/Query.API/API.IntegrationTests/SwaggerTests.SwaggerDoc_20230101_VerifiesFullBearerSecuritySchemeStructure.verified.txt (hunk 1 above)
Comment (sahma19) [reply domains/certificates/Query.API/API.IntegrationTests/SwaggerTests.SwaggerDoc_20230101_VerifiesFullBearerSecuritySchemeStructure.verified.txt:4]: I will check :)

Comment (sahma19) [reply domains/certificates/Query.API/API.IntegrationTests/SwaggerTests.SwaggerDoc_20230101_VerifiesFullBearerSecuritySchemeStructure.verified.txt:4]: Without /r + /n:

image


With:

image

You decide.

**Review by ckr123 — APPROVED** [review]

**Pull request merged**